### DISCUSSION TITLE
Brunch-Recommended Way of Setting User-Defined System-Wide Startup Script

### DISCUSSION THREAD

**Discussion by yasurok** [sebanc/brunch#1740] [Q&A]
What is the Brunch-recommended way to add a user-defined system-wide startup script? For starters, I would like to have the script [in this thread on activating the internal LTE modem](https://github.com/sebanc/brunch/issues/442#issuecomment-1336201815) run at every boot.

Thank you for your help in advance!

**Comment by sebanc** [comment]
Have a look here:
https://github.com/sebanc/brunch/blob/r107/brunch-patches/99-bootscripts.sh

I just noticed the comment is incorrect, the correct path for scripts is: /mnt/stateful_partition/brunch/bootscripts

**Reply by yasurok** [reply]
Thank you!!! I am very happy to report that Brunch executed the script as intended.
